src/pages/GameDetail.vue

```diff
@@ -14,7 +14,7 @@
                         <game-rating :rating="game.rating" />
 
                         <div class="actions" v-if="list.games.includes(game.id)">
-                            <button class="error small hollow" @click="removeGame">
+                            <button class="error small hollow" v-shortkey="['del']" @shortkey="removeGame" @click="removeGame">
```

Comment:
does this conflict with the game notes shortcut that uses the same key?

Reply:
It does not, but im thinking about moving delete actions to be shift + delete instead to stop accidentally hitting the delete key.